.github/workflows/ci_cron.yml

```diff
@@ -310,40 +304,40 @@ jobs:
         uses: ./.github/actions/annotate
 
 
-  notify_slack_fail:
-    name: Notify slack fail
-    needs: [unit_tests, multiverse, infinite_tracing]
-    runs-on: ubuntu-22.04
-    if: always()
-    steps:
-      - uses: technote-space/workflow-conclusion-action@v3
-      - uses: voxmedia/github-action-slack-notify-build@v1
-        if: ${{ env.WORKFLOW_CONCLUSION == 'failure' && github.event_name != 'workflow_dispatch' }}
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.RUBY_GITHUB_ACTIONS_BOT_WEBHOOK }}
-        with:
-          channel: ruby-agent-notifications
-          status: FAILED
-          color: danger
-
-
-  notify_slack_success:
-    name: Notify slack success
-    needs: [unit_tests, multiverse, infinite_tracing]
-    runs-on: ubuntu-22.04
-    if: always()
-    steps:
-      - uses: technote-space/workflow-conclusion-action@v3
-      - run: echo ${{ github.event_name }}
-      - uses: Mercymeilya/last-workflow-status@v0.3.2
-        id: last_status
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-      - uses: voxmedia/github-action-slack-notify-build@v1
-        if: ${{ env.WORKFLOW_CONCLUSION == 'success' && steps.last_status.outputs.last_status == 'failure' && github.event_name != 'workflow_dispatch' }}
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.RUBY_GITHUB_ACTIONS_BOT_WEBHOOK }}
-        with:
-          channel: ruby-agent-notifications
-          status: SUCCESS
-          color: good
+  # notify_slack_fail:
```

Comment:
I suppose we will want to reenable Slack notifications! I can make this PR if you want @tannalynn